txpool/txpool.go

```diff
@@ -724,90 +739,111 @@ func (p *TxPool) pruneAccountsWithNonceHoles() {
 // (only once) and an enqueueRequest is signaled.
 func (p *TxPool) addTx(origin txOrigin, tx *types.Transaction) error {
 	if p.logger.IsDebug() {
-		p.logger.Debug("add tx",
-			"origin", origin.String(),
-			"hash", tx.Hash.String(),
-		)
+		p.logger.Debug("add tx", "origin", origin.String(), "hash", tx.Hash.String())
 	}
 
 	// validate incoming tx
 	if err := p.validateTx(tx); err != nil {
 		return err
 	}
 
+	// calculate tx hash
+	tx.ComputeHash()
+
+	// initialize account for this address once or retrieve existing one
+	account, _ := p.createAccountOnce(tx.From)
```

Comment:
Why adding the bool return value if we're ignoring it?

Reply:
But we're not using it 🤔  why not remove the return value?

Reply:
fixed https://github.com/0xPolygon/polygon-edge/pull/1656/commits/01883acad54a6b729cac8b6914df46a1f678a01a